go-controller/pkg/util/util.go

```diff
@@ -323,3 +324,46 @@ func UpdateNodeSwitchExcludeIPs(nbClient libovsdbclient.Client, nodeName string,
 
 	return nil
 }
+
+// NetNameInfo is structure which holds network name information
+type NetNameInfo struct {
```

Comment:
I think all these structs / functions would be more suited to a separate helper file - utils.utils can be literally anything. It will become a junkyard of unrelated things eventually.

Would it make sense to move this to a multi-homing types / funcs in a separate pkg ? Or at least a multi-homing related utils ...

Reply:
ok